caikit/core/registries.py

```diff
@@ -0,0 +1,75 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Single spot where stick accessors for global shared state
+
+(Shining the light on the shared state so that we can clean it up and re-home it)
```

Comment:
Not sure this comment belongs in the mainline?

Reply:
Yeah... I was imagining that we wouldn't want to leave these here long term but if this doesn't feel that bad then we can 🔥 it down